sktime/forecasting/pytorchforecasting.py

```diff
@@ -308,7 +308,7 @@ def get_test_params(cls, parameter_set="default"):
         del parameter_set  # to avoid being detected as unused by ``vulture`` etc.
 
         try:
-            _check_soft_dependencies("pytorch_forecasting", severity="error")
+            _check_soft_dependencies("pytorch-forecasting", severity="error")
```

Comment:
noting: instead of this, you can do `severity="none"`, i.e., `if _check_soft_dependencies`, that will have the same effect without `try` / `except`

Reply:
(feel free to fix that in this PR)

Reply:
updated